entrypoint.sh

```diff
@@ -154,6 +154,7 @@ fi
 echo ::group::make test
 export CTEST_OUTPUT_ON_FAILURE=1
 cd "$GITHUB_WORKSPACE"/build
+. colcon_command_prefix_test.sh
```

Comment:
The only part of the code that uses colcon is when compiling dependencies from source, see line 93 above. And that's sourcing the workspace. I'm not sure what this line is supposed to be doing.

The PR you mentioned, https://github.com/ignitionrobotics/ign-sensors/pull/38, isn't installing any dependencies from source, so that build shouldn't need colcon at all.

Maybe what you need is a `make install` on `after_make.sh` like [this](https://github.com/ignitionrobotics/ign-gui/blob/03466fc66a781c048826fbb1075a697b8b85028a/.github/ci/after_make.sh#L6)?

Reply:
Line 93 only source the dependencies if you include some dependencies from source.

This file is only going to load the `package.sh` of the dependencies included in the `CMakeLists.txt`. An example:

```bash
# generated from colcon_core/shell/template/command_prefix.sh.em
. "/home/ahcorde/ignition_dome/install/share/ignition-cmake2/package.sh"
. "/home/ahcorde/ignition_dome/install/share/ignition-math6/package.sh"
. "/home/ahcorde/ignition_dome/install/share/ignition-plugin1/package.sh"
. "/home/ahcorde/ignition_dome/install/share/ignition-rendering4/package.sh"
```

This other approach may require:
  - include `-DCMAKE_INSTALL_PREFIX` to the cmake call
  - source this directoryif it's not default one
  - and `make install`
or
  - `make install`
  - setup manually `LD_LIBRARY_PATH` because make install will install this new library in `/usr/local` not in `/usr/` . [See here](https://github.com/ignitionrobotics/ign-gui/blob/03466fc66a781c048826fbb1075a697b8b85028a/.github/ci/after_make.sh#L9)